### PR TITLE
Restyle columns to single column

### PR DESCRIPTION
**Pull request by Richardson-e**

### CODE REVIEW

File: buildSrc/src/main/groovy/org/grails/plugin/PluginsPage.groovy [hunk 1]
```diff
@@ -218,17 +212,6 @@ class PluginsPage {
         MarkupBuilder mb = new MarkupBuilder(writer)
         mb.li(class: 'plugin') {
 
-//                ul(class: 'iconlinks') {
```
Comment (sdelamo) [inline buildSrc/src/main/groovy/org/grails/plugin/PluginsPage.groovy:221]: I would delete this commented out code.

**Review by sdelamo — COMMENTED** [review]
With this changes, the design creates a huge blank space 

IMG_7599

**Review by sdelamo — CHANGES_REQUESTED** [review]
Please, remove commented out code.

File: assets/javascripts/plugins-search.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+//    var label = element.childNodes[y].getElementsByClassName('labels');
+//    var liElements = label.getElementsByTagName('li')
+//        if (element.childNodes[y].className == "labels") {
+//
+//        //TODO GET LI FROM UL
+//            for (var i=0;y <liElements.length;i ++){
+//                labels.push(liElements[i])
+//            }
+//        }
+//    }
```
Comment (puneetbehl) [inline assets/javascripts/plugins-search.js:90]: Please delete this code.

**Review by puneetbehl — CHANGES_REQUESTED** [review]

**Pull request merged**